### PR TITLE
Remove duplicated code used by Toolbar functionnality

### PR DESCRIPTION
**Pull request by Fia3D**
I removed the code duplication I was using to make specific part working with the toolbar, and used the same as the base UI. 
I extracted some code to method when needed.

### CODE REVIEW

**Comment by Fayti1703** [comment]
This seems to do a *bit* more than the commit message & PR message indicate?

**Comment by Fia3D** [comment]
> This seems to do a _bit_ more than the commit message & PR message indicate?

It remove two files that was duplicated code that got edited to fit toolbar. 
Some other part have been extracted to their own method and moved from the removed files.

File: Ktisis/Interface/Components/ControlButtons.cs [hunk 1]
```diff
@@ -127,34 +127,28 @@ public static class ControlButtons {
...
-				else
-					if (ImGui.GetIO().KeyCtrl) {
-						Ktisis.Configuration.GizmoOp = Ktisis.Configuration.GizmoOp.ToggleFlag(OPERATION.ROTATE);
-						Ktisis.Configuration.GizmoOp = Ktisis.Configuration.GizmoOp.ToggleFlag(OPERATION.ROTATE_X);
-						Ktisis.Configuration.GizmoOp = Ktisis.Configuration.GizmoOp.ToggleFlag(OPERATION.ROTATE_Y);
-						Ktisis.Configuration.GizmoOp = Ktisis.Configuration.GizmoOp.ToggleFlag(OPERATION.ROTATE_Z);
-					}
+					Ktisis.Configuration.GizmoOp = ImGui.GetIO().KeyShift ? Ktisis.Configuration.GizmoOp.AddFlag(operation) : operation;
+				else if (ImGui.GetIO().KeyCtrl)
+					Ktisis.Configuration.GizmoOp = Ktisis.Configuration.GizmoOp.ToggleFlag(OPERATION.ROTATE, OPERATION.ROTATE_X, OPERATION.ROTATE_Y, OPERATION.ROTATE_Z);
```
Comment (Fayti1703) [inline Ktisis/Interface/Components/ControlButtons.cs:132]: And also does this

File: Ktisis/Interface/Components/ControlButtons.cs [hunk 2]
```diff
@@ -127,34 +127,28 @@ public static class ControlButtons {
...
-			if (operation == OPERATION.TRANSLATE) help += "Position";
-			else if (operation == OPERATION.ROTATE) help += "Rotation";
-			else if (operation == OPERATION.SCALE) help += "Scale";
-			else if (operation == OPERATION.UNIVERSAL) help += "Universal";
+			help += operation switch {
+				OPERATION.TRANSLATE => "Position",
+				OPERATION.ROTATE => "Rotation",
+				OPERATION.SCALE => "Scale",
+				var _ => throw new ArgumentOutOfRangeException(nameof(operation), operation, null)
+			};
```
Comment (Fayti1703) [inline Ktisis/Interface/Components/ControlButtons.cs:151]: And this

File: Ktisis/Interface/Components/ControlButtons.cs [hunk 3]
```diff
@@ -18,7 +18,7 @@
 
 namespace Ktisis.Interface.Components {
 	public static class ControlButtons {
-		public static Vector2 ButtonSize = new Vector2(ImGui.GetFontSize() * 1.6f) + ImGui.GetStyle().FramePadding;
+		public static Vector2 ButtonSize => new Vector2(ImGui.GetFontSize() * 1.6f) + ImGui.GetStyle().FramePadding;
```
Comment (Fayti1703) [inline Ktisis/Interface/Components/ControlButtons.cs:21]: And this

File: Ktisis/Interface/Windows/Toolbar/AdvancedWindow.cs [hunk 4]
```diff
@@ -9,28 +9,22 @@
 
 namespace Ktisis.Interface.Windows.Toolbar {
 	public static class AdvancedWindow {
-		private static bool Visible = false;
-
-		public static Vector4 ColGreen = new(0, 255, 0, 255);
-		public static Vector4 ColRed = new(255, 0, 0, 255);
-
-		public static TransformTable Transform = new();
+		private static bool _visible;
```
Comment (Fayti1703) [inline Ktisis/Interface/Windows/Toolbar/AdvancedWindow.cs:12]: And renames this field

Comment (Fia3D) [reply Ktisis/Interface/Windows/Toolbar/AdvancedWindow.cs:12]: Ah yeah I was working on refactoring on different local branch dunno how it came here ^^

**Comment by Maxunit** [comment]
I merged the toolbar/reduce-duplicate-code branch into my testing branch yesterday and haven't come across any issues with the Toolbar, which I use all the time. Every feature works as intended (so far) but I will continue derping around.

**Review by Fayti1703 — APPROVED** [review]

**Pull request merged**